index.bs

```diff
@@ -4816,21 +4816,22 @@ let loopEnd;
 let playbackRate;
 
 // Variables for the node's playback parameters
-let start = 0, offset = 0; // Set by start()
-let stop = Infinity; // Set by stop(), or by start() with a supplied duration
+let start = 0, offset = 0, duration = Infinity; // Set by start()
+let stop = Infinity; // Set by stop()
+
 
 // Variables for tracking node's playback state
 let bufferTime = 0, started = false, enteredLoop = false;
 let dt = 1 / context.sampleRate;
 
 // Handle invocation of start method call
-function handleStart(when, pos, duration) {
+function handleStart(when, pos, dur) {
 	if (arguments.length >= 1) {
 		start = when;
 	}
 	offset = pos;
 	if (arguments.length >= 3) {
-		stop = when + duration;
+		duration = dur;
```

Comment:
Need to define `duration`?

Reply:
The `duration` variable is already defined, on line 4819 (same place as `offset`).

Reply:
Ack.